sphinx/transforms/__init__.py

```diff
@@ -404,6 +405,22 @@ def apply(self, **kwargs: Any) -> None:
                 info = r.groupdict()
             node.attributes.update(info)
 
+class GlossarySorter(SphinxTransform):
+    """Sort glossaries that have the ``sorted`` flag."""
+    # Must be done after i18n.
+    default_priority = 30
```

Comment:
IMO, this transform has too much high priority. How about changing this to the middle priority 500?
https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_transform